views/homepage.ejs

```diff
@@ -6,276 +6,297 @@
-      map.addLayer({'id': 'keyboardnav','type': 'line','source': 'keyboardsquare','layout': {},'paint': {'line-color': '#fff','line-opacity': 1}});
-        window.addEventListener("keydown", dealWithKeydown, false);
-        window.addEventListener("keyup", dealWithKeyup, false);
-      })
-    function moveRedSquare(e){
-      var b = map.getBounds();
-      // console.log("bounds = ",b)
-      minimap.getSource("redsquare").setData({
-            'type': 'Feature',
-            'properties': {
-                'name': 'redsquare'
-            },
-            'geometry': {
-                'type': 'Polygon',
-                'coordinates':[[
-                [b._sw.lng, b._sw.lat],
-                    [b._ne.lng, b._sw.lat],
-                    [b._ne.lng, b._ne.lat],
-                    [b._sw.lng, b._ne.lat],
-                    [b._sw.lng, b._sw.lat]
-                    ]]
-            }
-        })
-      // console.log(minimap.getSource('redsquare'))
+// mapboxgl.accessToken = 'pk.eyJ1IjoiY2FtaWthemVncmVlbiIsImEiOiJINE9RRFhvIn0.K7qZW250wpNclZ0Ii-EG6Q';
+// $(document).ready(function(){
+//   // var url = "https://api.mapbox.com/styles/v1/camikazegreen/cirhvvqh00015hhmcwbgi25vl/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoiY2FtaWthemVncmVlbiIsImEiOiJINE9RRFhvIn0.K7qZW250wpNclZ0Ii-EG6Q"
+//   var url = 'mapbox://styles/camikazegreen/cirhvvqh00015hhmcwbgi25vl'
 
-    }
-    function moveKeyboardSquare(e){
-      //check the center of the current square, and only move if it is outside of the bounds
-      //get the center of the square
-      //make sure the lat is not in between the bounds
-      if(currentSquare.center[1]<currentBounds._sw.lat || currentSquare.center[1]>currentBounds._ne.lat || currentSquare.center[0]<currentBounds._sw.lng || currentSquare.center[0]>currentBounds._ne.lng){
-          var b = map.getCenter();
-      // console.log("bounds = ",b.lat)
-      //Adjust to nearest latlng on the grid
-      var blng = b.lng;
-          blng = 25*Math.round(blng*40)/1000;
-      var blat = b.lat;
-          blat = 2*Math.round(blat*50)/100;
-          currentSquare.center = [blng+0.0125,blat+0.01]
-          updateDataBox();
-          currentSquare.data = {
-            'type': 'Feature',
-            'properties': {
-                'name': 'keyboardsquare'
-            },
-            'geometry': {
-                'type': 'Polygon',
-                'coordinates':[[
-                [blng, blat],
-                    [blng+0.025, blat],
-                    [blng+0.025, blat+0.02],
-                    [blng, blat+0.02],
-                    [blng, blat]
-                    ]]
-            }
-        }
-      map.getSource("keyboardsquare").setData(currentSquare.data)
+//   //whole map bounds: {"_southWest":{"lat":33.24787594792436,"lng":-17.314453125},"_northEast":{"lat":54.265224078605655,"lng":42.71484375}}
```

Comment:
If you want to save for later, you can delete out of here , create a new branch and add it then do another pull request